src/emulator/renderer/src/load_shaders.cpp

```diff
@@ -5,6 +5,7 @@
 
 #include <gxm/functions.h>
 #include <gxm/types.h>
+#include <shadergen/functions.h>
 #include <util/log.h>
 
 #include <fstream>
```

Comment:
Are any includes now unnecessary? Quite a lot of code was deleted.

Reply:
`gxm/functions.h` was not needed